### PR TITLE
fix(validator): Forbidden rule does not support float

### PR DESCRIPTION
**Pull request by safaozturk93**
#436

### CODE REVIEW

File: cerberus/errors.py [hunk 1]
```diff
@@ -70,7 +70,7 @@
 # TODO remove KEYSCHEMA AND VALUESCHEMA with next major release
 KEYSRULES = KEYSCHEMA = ErrorDefinition(0x83, 'keysrules')
 VALUESRULES = VALUESCHEMA = ErrorDefinition(0x84, 'valuesrules')
-BAD_ITEMS = ErrorDefinition(0x8f, 'items')
+BAD_ITEMS = ErrorDefinition(0x8F, 'items')
```
Comment (funkyfuture) [inline cerberus/errors.py]: that is an unnecessary change.

File: cerberus/validator.py [hunk 2]
```diff
@@ -1189,6 +1189,11 @@ def _validate_forbidden(self, forbidden_values, field, value):
         elif isinstance(value, int):
             if value in forbidden_values:
                 self._error(field, errors.FORBIDDEN_VALUE, value)
+        elif isinstance(value, float):
```
Comment (funkyfuture) [inline cerberus/validator.py]: please keep it simple. afaict, there's no need to branch off for float or integer values, only for container values that are not strings.

File: cerberus/errors.py (hunk 1 above)
Comment (safaozturk93) [reply cerberus/errors.py]: this violates black reformatting rule in travis pipeline.

File: cerberus/validator.py (hunk 2 above)
Comment (safaozturk93) [reply cerberus/validator.py]: I will look at it.

**Comment by funkyfuture** [comment]
Closing in favour of #455.
